### ISSUE TITLE
scale up machine to more CPUs and bandwidth

### ISSUE THREAD

**Issue by kylemcdonald** [kylemcdonald/HowWeActTogether-Tracking#37]

**Comment by kylemcdonald** [comment]
scaled from 512MB RAM / 1 CPU / 1 TB transfer at $0.007/hr to 16GB RAM / 8 CPUs / 6 TB transfer at $0.238/hr.

resizing took about 5 minutes of downtime. we can make it bigger if we need to.

but i checked the CPU and bandwidth usage before restarting, and i saw only a 7% increase between min and max CPU usage, which would put us at about 14 users per CPU, or about 110 simultaneous users. and 0.6Mbps outbound 0.2Mbps inbound peaks, which is harder to work out... but if we used all our 6TB in one week, that would come out to a sustained ~74Mbps, which comes out to about 92 simultaneous users.

in short, right now we are set for ~100 simultaneous streams for a week. if there is a peak of more, we will fail due to CPU.
